src/test/e2e/helpers/test-helper.ts

```diff
@@ -292,11 +292,12 @@ export async function setupAppWithCustomAuth(
 
 export async function setupAppWithBaseUrl(
     stores: IUnleashStores,
+    baseUriPath = '/hosted',
```

Comment:
This is for convenience. Being able to choose the base path makes sense in the openapi tests, so this seems like a good choice to me.